src/components/pages/ProductListPage/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { useSelector } from "react-redux";
+
+import { getProductList } from "../../../reducers/productList";
+
+import Spinner from "../../common/Spinner";
+import ProductCard from "./ProductCard";
+import GridContainer from "../../common/GridContainer";
+import ErrorPage from "../ErrorPage";
+import { useThunk } from "../../../hooks/useThunk";
+
+function ProductListPage() {
+  const productList = useSelector((state) => state.productList.data);
+  const isLoading = useSelector((state) => state.productList.isLoading);
+  const errorMessage = useSelector((state) => state.productList.errorMessage);
```

Comment:
```suggestion
  const { isLoading, errorMessage, data: productList} = useSelector((state) => state.productList);
```
state를 모두 사용하고있는데, 이렇게 줄여보는건 어떨까요?